docker/hadoop/common/startup/50_hive_init

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+set -e -x
+
+sudo -E -u hdfs hadoop fs -mkdir -p /user/hive/warehouse
+sudo -E -u hdfs hadoop fs -chmod a+w /user/hive/warehouse
+sudo -E -u hdfs hadoop fs -chmod a+w /tmp
+
+# init mysql database for metastore
+
+rm -rf /var/lib/mysql
```

Comment:
I think this should be moved to a separate file like `..._mysql_init`.

Reply:
will do